<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XOR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelLog">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Логи</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserLog">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>311</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditInputMask">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOutputPath">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelInputMask">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Маска входных файлов: (*.&quot;*&quot;):</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutputPath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Путь для сохранения файлов:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxDeleteAfterProcessing">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>331</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалять оригинальные файлы</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxFileConflict">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Действия при совпадении имен:</string>
    </property>
    <widget class="QRadioButton" name="radioButtonOverwrite">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>281</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Перезаписать</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonAddCounter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>311</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить счетчик к имени</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxRunMode">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Режим запуска:</string>
    </property>
    <widget class="QRadioButton" name="radioButtonSingleRun">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>221</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Разовый запуск</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonTimerRun">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>221</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>По таймеру</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelInterval">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Периодичность опроса (мс):</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxInterval">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>350</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>100</number>
    </property>
    <property name="maximum">
     <number>99000</number>
    </property>
    <property name="value">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelXorValue">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Значение XOR (8 байт, HEX):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditXorValue">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>380</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStart">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Старт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStop">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>420</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Стоп</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
